websites/C/CHub/presence.ts

```diff
@@ -0,0 +1,122 @@
+			"/ranking": { state: "Viewing Rankings" },
+			"/application/new": { state: "Creating an Application" },
+			"/application/my": { state: "Viewing Applications" },
+			"/application/all": { state: "Viewing All Applications" },
+			"/manage/user": { state: "Viewing Pending Users" },
+			"/audit": { state: "Viewing Audit Log" },
+			"/config": { state: "Editing Configuration" },
+			"/settings": { state: "Updating Settings" },
+			"/notification": { state: "Viewing Notifications" },
+		},
+		appages: Record<string, PresenceData> = {
+			"/": { details: "Viewing CHub" },
+			"/features": { details: "Viewing Features" },
+			"/subscribe": { details: "Subscribing to CHub" },
+			"/setup": { details: "Creating a Drivers Hub" },
+		},
+		{ title: pstate } = document;
+	switch (host) {
+		case "drivershub.charlws.com":
+			for (const [p, data] of Object.entries(appages))
+				if (path.includes(p)) presenceData = { ...presenceData, ...data };
+			presenceData.buttons = [
+				{ label: "CHub", url: "https://drivershub.charlws.com" },
+				{ label: "Features", url: "https://drivershub.charlws.com/features" },
+			];
+			break;
+		case "map.charlws.com":
+			presenceData.details = "Viewing Map";
+			presenceData.state = pstate;
+			presenceData.buttons = [{ label: "View Map", url: document.URL }];
+			break;
+		case "wiki.charlws.com":
+			presenceData.details = "Viewing CHub Wiki";
+			presenceData.state = pstate;
+			presenceData.buttons = [{ label: "View CHub Wiki", url: document.URL }];
+			break;
+	}
+	if (!host.endsWith(".charlws.com")) {
+		for (const [p, data] of Object.entries(dhpages))
+			if (path.includes(p)) presenceData = { ...presenceData, ...data };
+		switch (true) {
+			case path.includes("/delivery/"):
+				presenceData.details = `Viewing Delivery #${
+					path.split("/")[path.split("/").length - 1]
+				}`;
+				if (
+					document.querySelector("#delivery-detail-source-city").textContent ===
+					""
+				)
+					presenceData.state = "Loading...";
+				else {
+					presenceData.state = `${
+						document.querySelector("#delivery-detail-source-city").textContent
+					} -> 
+						${document.querySelector("#delivery-detail-destination-city").textContent}`;
+				}
+				presenceData.buttons = [{ label: "View Delivery", url: document.URL }];
+				break;
+			case path.includes("/member/"):
+				presenceData.details = "Viewing Profile";
+				presenceData.state = document.querySelector(
+					"#profile-info > h1 > b"
+				).innerHTML;
+				presenceData.buttons = [{ label: "View Profile", url: document.URL }];
+				break;
+		}
+	}
+
+	const Domain2Abbr: { [key: string]: string } = {
+		"drivershub.charlws.com": "chub",
+		"hub.atmvtc.com": "atm",
+		"hub.gokboru.net.tr": "gokboru",
+		"hub.lsavtc.com": "lsa",
+		"hub.globaltrucking.uk": "gt",
+		"hub.mdlogistics.ga": "md",
+		"drivershub.ets2mcg.org": "ets2mcg",
+	};
+	if (Object.keys(Domain2Abbr).includes(host))
+		presenceData.largeImageKey = Domain2Abbr[host];
```

Comment:
Change it to imgur links.
```suggestion
		presenceData.largeImageKey = Domain2Abbr[host];
```

Reply:
I'm confused, what is the suggested change supposed to refer to?